data-prepper-plugins/s3-source/src/main/java/org/opensearch/dataprepper/plugins/source/s3/S3InputFile.java

```diff
@@ -46,6 +47,15 @@ public long getLength() {
         return getMetadata().contentLength();
     }
 
+    /**
+     * Return the last modified time of the file
+     *
+     * @return last modified time
+     */
+    public Instant getLastModified() {
+        return getMetadata().lastModified();
```

Comment:
Are you sure this will return the same last modified time from S3?

Reply:
Yes.